### PR TITLE
Fix resize of appointment with custom view startDayHour & endDayHour (T804779)

### PR DESCRIPTION
**Pull request by Olexi**

### CODE REVIEW

File: testing/functional/tests/scheduler/resizeAppointments/basic.ts [hunk 1]
```diff
@@ -31,7 +31,11 @@ fixture `Resize appointments in the Scheduler basic views`
         .expect(resizableAppointment.date.startTime).eql(`10:00 AM`)
         .expect(resizableAppointment.date.endTime).eql(`11:00 AM`);
 
-}).before(() => createScheduler(view, dataSource)));
+}).before(() => createScheduler({
+    views: [view],
```
Comment (naralewinsky) [inline testing/functional/tests/scheduler/resizeAppointments/basic.ts]: What do you think about removing this argument?
'''
views: [view],
'''

File: testing/functional/tests/scheduler/resizeAppointments/basic.ts [hunk 2]
```diff
@@ -58,4 +62,8 @@ test(`Resize in the "month" view`, async t => {
         .expect(resizableAppointment.date.startTime).eql(`10:00 AM`)
         .expect(resizableAppointment.date.endTime).eql(`11:00 AM`);
 
-}).before(() => createScheduler('month', dataSource));
+}).before(() => createScheduler({
+    views: ['month'],
```
Comment (ghost) [inline testing/functional/tests/scheduler/resizeAppointments/basic.ts:66]: What do you think about removing this **'views: [view]'** argument?

File: testing/functional/tests/scheduler/resizeAppointments/timeline.ts [hunk 3]
```diff
@@ -58,4 +62,31 @@ test(`Resize in the "timelineMonth" view`, async t => {
         .expect(resizableAppointment.date.startTime).eql(`10:00 AM`)
         .expect(resizableAppointment.date.endTime).eql(`11:00 AM`);
 
-}).before(() => createScheduler('timelineMonth', dataSource));
+}).before(() => createScheduler({
+    views: ['timelineMonth'],
```
Comment (ghost) [inline testing/functional/tests/scheduler/resizeAppointments/timeline.ts:66]: What do you think about removing this **'views: [view]'** argument?

File: testing/functional/tests/scheduler/resizeAppointments/basic.ts (hunk 2 above)
Comment (Olexi) [reply testing/functional/tests/scheduler/resizeAppointments/basic.ts:66]: I think that tests must satisfy a "Single Responsibility Principle", which says that any function must serve only one purpose. In that case, test will be "broken" if someone will break a logic referring to `views`. So, the test will `false-trigger` on the problem which has no common with `Resizing`

File: testing/functional/tests/scheduler/resizeAppointments/timeline.ts (hunk 3 above)
Comment (Olexi) [reply testing/functional/tests/scheduler/resizeAppointments/timeline.ts:66]: See above.

File: testing/functional/tests/scheduler/resizeAppointments/basic.ts (hunk 2 above)
Comment (ghost) [reply testing/functional/tests/scheduler/resizeAppointments/basic.ts:66]: I understood.

**Review by ghost — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Pull request merged**